dipy/tracking/tests/test_fbc.py

```diff
@@ -0,0 +1,47 @@
+from dipy.denoise.enhancement_kernel import EnhancementKernel
+from dipy.tracking.fbcmeasures import FBCMeasures
+
+from dipy.viz import fvtk
+from dipy.viz.colormap import line_colors
+from dipy.viz import window, actor
+
+import numpy as np
+import numpy.testing as npt
+
+def test_fbc():
+    """Test the FBC measures on a set of fibers"""
+
+    # Generate two fibers of 10 points
+    streamlines = []
+    for i in range(2):
+        fiber = np.zeros((10,3))
+        for j in range(10):
+            fiber[j,0] = j
+            fiber[j,1] = i*0.2
+            fiber[j,2] = 0
+            streamlines.append(fiber)
+
+    # Create lookup table. Set seed for deterministic results.
+    D33 = 1.0
+    D44 = 0.04
+    t = 1
+    np.random.seed(1)
```

Comment:
The randomness in this old version was inside EnhancementKernel, in which orientations were randomly chosen and electrostatic repulsion was used to distribute the orientations evenly on the sphere. However this game some issues with the bots, showing different outcomes despite setting the seed. This was fixed by giving the kernel a small set of orientations manually.